src/core/tests/Test_CoreMain.py

```diff
@@ -1143,6 +1168,624 @@ def __check_telemetry_events(self, runtime):
+        self.assertEqual(assessment_started_by, truncated_assessment_started_by)
+
+        # Assert installation truncation
+        installation_truncated_substatus = substatus_file_data[0]["status"]["substatus"][1]
+        self.__assert_truncated_status_output(installation_truncated_substatus, Constants.PATCH_INSTALLATION_SUMMARY, Constants.STATUS_WARNING)
+        # Assert installation tombstone
+        self.__assert_truncated_installation_tombstone(installation_truncated_substatus, patch_count_for_installation + 2)
+        # Assert installation truncated error
+        self.__assert_truncated_error(substatus_file_data[0]["status"]["substatus"][0], error_count=0)
+
+        truncated_installation_msg = json.loads(installation_truncated_substatus["formattedMessage"]["message"])
+        truncated_installation_activity_id = truncated_installation_msg['installationActivityId']
+        truncated_installation_reboot_status = truncated_installation_msg['rebootStatus']
+        truncated_installation_maintenance_window = truncated_installation_msg['maintenanceWindowExceeded']
+        truncated_installation_not_selected = truncated_installation_msg['notSelectedPatchCount']
+        truncated_installation_excluded = truncated_installation_msg['excludedPatchCount']
+        truncated_installation_pending = truncated_installation_msg['pendingPatchCount']
+        truncated_installation_installed = truncated_installation_msg['installedPatchCount']
+        truncated_installation_failed = truncated_installation_msg['failedPatchCount']
+        truncated_installation_start_time = truncated_installation_msg['startTime']
+        truncated_installation_last_modified_time = truncated_installation_msg['lastModifiedTime']
+        truncated_installation_maintenance_id = truncated_installation_msg['maintenanceRunId']
+
+        # validate all installation other fields in the message object are equal in both status files
+        self.assertEqual(installation_activity_id, truncated_installation_activity_id)
+        self.assertEqual(installation_reboot_status, truncated_installation_reboot_status)
+        self.assertEqual(installation_maintenance_window, truncated_installation_maintenance_window)
+        self.assertEqual(installation_not_selected, truncated_installation_not_selected)
+        self.assertEqual(installation_excluded, truncated_installation_excluded)
+        self.assertEqual(installation_pending, truncated_installation_pending)
+        self.assertEqual(installation_installed, truncated_installation_installed)
+        self.assertEqual(installation_failed, truncated_installation_failed)
+        self.assertEqual(installation_start_time, truncated_installation_start_time)
+        self.assertEqual(installation_last_modified_time, truncated_installation_last_modified_time)
+        self.assertEqual(installation_maintenance_id, truncated_installation_maintenance_id)
```

Comment:
I feel like I've left this comment in some earlier version of a PR on this -- why do lines 1394 to 1405 exist? The variable assignments simply add lines of code and (minor) use additional memory allocation. All this can be reduced directly into what's 1407 to 1418. 

This pattern is in several portions of this test code and is verbosity that needn't be there.

@rane-rajasi to review for this everywhere.

Reply:
good catch here, i've should reduce the # of variables. these assertions are added in response to one of rajasi's older comments, she wants to have more confidence that the message json fields are the same except for patches and error. addressed

Reply:
Verified addressed.